include/inexor/vulkan-renderer/wrapper/descriptor.hpp

```diff
@@ -29,7 +29,7 @@ class ResourceDescriptor {
     /// @param descriptor_writes The write descriptor sets.
     /// @param name The internal debug marker name of the resource descriptor.
     ResourceDescriptor(const Device &device, std::uint32_t swapchain_image_count,
-                       std::initializer_list<VkDescriptorType> pool_types,
+                       const std::vector<VkDescriptorType> &pool_types,
```

Comment:
Maybe we should change this back to `std::initializer_list`. Otherwise we can't inline the type of the resource descriptor, but we must pass a vector instead. We can somehow convert a std::vector to std::initializer_list in the descriptor builder.

Reply:
Oh wait.. `pool_types` isn't even required anymore!